src/Core/Application/Dependency/Repository/WorkflowRepositoryInterface.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Core\Application\Dependency\Repository;
+
+use SprykerSdk\SdkContracts\Entity\WorkflowInterface;
+
+interface WorkflowRepositoryInterface
+{
+    /**
+     * @param string $project
+     *
+     * @return array<\SprykerSdk\SdkContracts\Entity\WorkflowInterface>
+     */
+    public function findWorkflows(string $project): array;
```

Comment:
I would like to keep with Spryker convention here. we should use `find*` if we can get nullable result, but get if we get any result in any case.
For methods where we get array in any case it must be get*

Reply:
Fixed